src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -155,16 +156,23 @@ public ResponseEntity<Participation> startParticipation(@PathVariable Long exerc
         log.debug("REST request to start Exercise : {}", exerciseId);
         Exercise exercise = exerciseRepository.findByIdElseThrow(exerciseId);
         User user = userRepository.getUserWithGroupsAndAuthorities();
-        Participant participant = user;
         authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.STUDENT, exercise, user);
 
         // if the user is a student and the exercise has a release date, they cannot start the exercise before the release date
-        if (exercise.getReleaseDate() != null && exercise.getReleaseDate().isAfter(now())) {
```

Comment:
I don't see the reason behind this change as you still compare the start date afterwards.

Reply:
getIndividualReleaseDate also takes the StartDate into account

Reply:
Exactly. If it gets taken into account, why do we need to check the startDate afterwards?